ui/app/mirrors/edit/[mirrorId]/cdcGraph.tsx

```diff
@@ -13,9 +13,12 @@ import aggregateCountsByInterval from './aggregatedCountsByInterval';
 
 const aggregateTypeMap: { [key: string]: string } = {
```

Comment:
Can this key become a type? Would then be value for `aggregateCountsByInterval`'s `interval`

Reply:
Turns out this map is now unused so I've removed it